.github/workflows/build-and-release-stable.yml

```diff
@@ -140,49 +140,48 @@ jobs:
           version: latest
           args: release --clean --skip=validate
       - name: bump homebrew-core formula
-        uses: mislav/bump-homebrew-formula-action@v2
+        uses: mislav/bump-homebrew-formula-action@v3
```

Comment:
also bump it to use nodejs 20 runtime with v3

Reply:
Good call. 👍